pkg/BUILD

```diff
@@ -58,11 +58,10 @@ py_binary(
     srcs = ["make_rpm.py"],
     python_version = "PY3",
     srcs_version = "PY3",
-    # TODO(aiuto): Enable this for bazel 4.x
-    #target_compatible_with = select({
-    #    "//toolchains:have_rpmbuild": [],
-    #    "//conditions:default": [":not_compatible"],
-    #}),
+    target_compatible_with = select({
```

Comment:
I feel this really should be `exec_compatible_with`, but that's apparently not a configurable attribute.  Might be worth adding a comment here?

I suppose this goes back to the point of being able to define platform constraints for the `exec` configuration dynamically.

Reply:
Well. Maybe I can make that work.   I'll to some playing with it.